client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -98,6 +100,7 @@ function ReaderSubscriptionListItem( {
 					className="reader-subscription-list-item__site-url"
 				>
 					{ formatUrlForDisplay( siteUrl ) }
+					{ feed && feed.last_update && ( ' ' + translate( 'updated' ) + ' ' + lastUpdatedDate ) }
```

Comment:
this should use a placeholder anyway. `translate( 'updated {{lastUpdatedDate/}}', { components: { lastUpdatedDate } }`